app.R

```diff
@@ -10,6 +10,8 @@ library(magrittr)
 library(rgdal)
 library(DT)
 library(leaflet)
+library(remotes)
+remotes::install_github("ropensci/rgbif")
```

Comment:
I don't think it's good to directly install packages inside the app or server files - I think it really slows down loading the app. 

I **think** you can safely delete this, as when you deploy from RStudio it uses the version of the package currently installed on your system.